nireports/reportlets/utils.py

```diff
@@ -47,6 +46,8 @@
 from nibabel.spatialimages import SpatialImage
 from svgutils.transform import SVGFigure
 
+from nireports.tests import compression_missing_msg, have_compression
```

Comment:
We should keep some separation and avoid importing from the tests module for the package source - perhaps these convenience checks can be moved

Reply:
Agreed. Relocated in 062657c. My bad. :man_facepalming:.